doc/source/selections.rst

```diff
@@ -0,0 +1,440 @@
+    are returned intersected with the atoms of that instance.  Prefixing a
+    selection term with ``global`` causes its selection to be returned in its
+    entirety.  As an example, the ``global`` keyword allows for
+    ``lipids.select_atoms("around 10 global protein")`` --- where ``lipids`` is
+    a group that does not contain any proteins. Were ``global`` absent, the
+    result would be an empty selection since the ``protein`` subselection would
+    itself be empty.  When issuing
+    :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` from a
+    :class:`~MDAnalysis.core.universe.Universe`, ``global`` is ignored.
+
+.. deprecated:: 0.11
+   The use of ``fullgroup`` has been deprecated in favor of the equivalent
+   ``global group``.
+
+Dynamic selections
+==================
+
+By default :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` returns an
+:class:`~MDAnalysis.core.groups.AtomGroup`, in which the list of atoms is
+constant across trajectory frame changes. If
+:meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` is invoked with named
+argument ``updating`` set to ``True``, an
+:class:`~MDAnalysis.core.groups.UpdatingAtomGroup` instance will be returned
+instead. It behaves just like an :class:`~MDAnalysis.core.groups.AtomGroup`
+object, with the difference that the selection expressions are re-evaluated
+every time the trajectory frame changes (this happens lazily, only when the
+:class:`~MDAnalysis.core.groups.UpdatingAtomGroup` object is accessed so that
+there is no redundant updating going on)::
+
+ # A dynamic selection of corner atoms:
+ >>> ag_updating = universe.select_atoms("prop x < 5 and prop y < 5 and prop z < 5", updating=True)
+ >>> ag_updating
+ <UpdatingAtomGroup with 9 atoms>
+ >>> universe.trajectory.next()
+ >>> ag_updating
+ <UpdatingAtomGroup with 14 atoms>
+
+Using the ``group`` selection keyword for
+:ref:`preexisting-selections <pre-selections-label>`, one can
+make updating selections depend on
+:class:`~MDAnalysis.core.groups.AtomGroup`, or even other
+:class:`~MDAnalysis.core.groups.UpdatingAtomGroup`, instances.
+Likewise, making an updating selection from an already updating group will
+cause later updates to also reflect the updating of the base group::
+
+ >>> chained_ag_updating = ag_updating.select_atoms("resid 1:1000", updating=True)
+ >>> chained_ag_updating
+ <UpdatingAtomGroup with 3 atoms>
+ >>> universe.trajectory.next()
+ >>> chained_ag_updating
+ <UpdatingAtomGroup with 7 atoms>
+
+Finally, a non-updating selection or a slicing/addition operation made on an
+:class:`~MDAnalysis.core.groups.UpdatingAtomGroup` will return a static
+:class:`~MDAnalysis.core.groups.AtomGroup`, which will no longer update
+across frames::
+
+ >>> static_ag = ag_updating.select_atoms("resid 1:1000")
+ >>> static_ag
+ <UpdatingAtomGroup with 3 atoms>
+ >>> universe.trajectory.next()
+ >>> static_ag
+ <UpdatingAtomGroup with 3 atoms>
+
+.. _instance-selectors:
```

Comment:
Remove the whole section on instant selectors. https://github.com/MDAnalysis/mdanalysis/issues/1377